src/components/ProfilePage/ProfilePage.jsx

```diff
@@ -250,7 +273,21 @@ export function ProfilePage() {
                 <TalentData {...propsTalentData} ref={talentDataRef} />
                 <About {...propsAbout} ref={aboutRef} />
             </div>
+            <div className={s.proofs_side}>
+                <div className={s.updating_proofs}>
+                    <img
+                        className={`${s.add} ${active ? s.rotated : ""}`}
+                        onClick={() => {
+                            setActive((prev) => !prev);
+                        }}
+                        src={plus}
+                    ></img>
+                </div>
 
+                {active && (
+                    <AddingProofsForm active={active} setActive={setActive} />
```

Comment:
These props don't correspond to received in component
```suggestion
                    <AddingProofsForm active={active} setActive={setActive}  user={user} token={token}/>
```